src/update-all/update-all-dialog.ts

```diff
@@ -23,6 +23,6 @@ class ESPHomeUpdateAllDialog extends LitElement {
 
 declare global {
   interface HTMLElementTagNameMap {
-    "esphome-update-all-dialog": ESPHomeUpdateAllDialog;
+    "esphome-update-all-process-dialog": ESPHomeUpdateAllDialog;
```

Comment:
revert this. it doesn't match the custom element decorator.

Reply:
done